### DISCUSSION TITLE
Wiederherstellung von gesicherten Freetz Einstellungen

### DISCUSSION THREAD

**Discussion by ghost** [Freetz-NG/freetz-ng#146] [Off-topic]
Ich verwende derzeit einen Git Stand von Mitte November. Unfreiwilliger Weise habe ich meine Box recovern müssen. Nun wollte ich meine gesicherten Freetz Einstellungen wiederherstellen. Dies wird im WebIF als erfolgreich angezeigt. Nach dem durchgeführten Neustart sind aber keine der gesicherten Einstellungen im WebIF sichtbar.

**Hinweis:** Wie bei der Trunk Version erhalte ich auch bei der Tag Version Bootloops. Den Watchdog kann ich nicht deaktivieren, das diese Option vom Developer Status abhängig ist, welcher bei der Tag Version scheinbar nicht verfügbar ist. Nach Patchen der Konfiguration konnte ich den Watchdog deaktivieren. Nach dem Start der Box bin ich aber nicht auf das WebIF von Freetz gekommen.

**Comment by fda77** [comment]
Hier hast du schon die gleiche Frage gestellt und Antworten bekommen: https://www.ip-phone-forum.de/threads/flashen-über-bootloader-funktioniert-nicht-bootloader-flashen-gesperrt.309039
Man sollte nicht an verschiedenen Stellen über das gleiche diskutieren, daher DUPLICATE

Evtl wurde das auch durch diesen Fehler ausgelöst https://github.com/Freetz-NG/freetz-ng/issues/153
Bei mir funktioniert die Wiederherstellung übrigens

Manche Fritzboxen können auch "Dateien" unter /var/flash speichern. Daher gibt es die EXPERIMENTELLEN Parameter `modsave char` und `modsave file`

**Comment by ghost** [comment]
freetz-ng-17426M-68d7a170b: Komplett anderes Fehlerbild! Änderung von Einstelllungen im WebIF werden persitent gespeichert. Nur die Wiederherstellung funktioniert nicht. Temporär geändert - keine Änderung.

Tagged Version: Ich hatte das Standard WebIF Passwort drin und trotzdem kein WebIF.

**Comment by fda77** [comment]
Wie du selbst festgestellt hast liegt's mal wieder am Addon der DEBen:
 - https://www.digital-eliteboard.com/threads/oscam-addon-1-5-1-verhindert-wiederherstellung-der-freetz-einstellungen.496451/
 - https://www.ip-phone-forum.de/threads/309039
 - https://freetz-ng.github.io/freetz-ng/ADDONS.html
 - https://github.com/Freetz-NG/freetz-ng/commit/4071e9375ebed0f7b55f8d873d4a52b8c01b8774
